mobilecoind/src/sync.rs

```diff
@@ -589,4 +589,66 @@ mod test {
 
         assert!(!utxos.contains(&first_utxo));
     }
+
+    #[test_with_logger]
+    fn test_utxo_value_zero(logger: Logger) {
+        let mut rng: StdRng = SeedableRng::from_seed([98u8; 32]);
+
+        let account_keys: Vec<_> = (0..5).map(|_i| AccountKey::random(&mut rng)).collect();
+
+        let data = MonitorData::new(
+            account_keys[0].clone(),
+            DEFAULT_SUBADDRESS_INDEX, // first subaddress
+            5,                        // number of subaddresses
```

Comment:
Should this be 1? I see only the default subaddresses are used.

Reply:
Could be 1, could be 5 - doesn't make much of a difference for the test